parachain/pallets/outbound-queue/src/send_message_impl.rs

```diff
@@ -83,3 +83,12 @@ impl<T: Config> SendMessage for Pallet<T> {
 		Ok(ticket.id)
 	}
 }
+
+impl<T: Config> SendMessageFee for Pallet<T> {
+	type Balance = T::Balance;
+
+	/// Calculate fee in native currency for processing a message locally
```

Comment:
```suggestion
	/// The local component of the message processing fees in native currency
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/999/commits/5c03ad858d2f4c046b7dc26ce9cd6c1e5d615a34